apis/dbaas.redhat.com/v1alpha1/crunchybridgeinventory_types.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	dbaasv1alpha1 "github.com/RHEcosystemAppEng/dbaas-operator/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// CrunchyBridgeInventory is the Schema for the crunchybridgeinventories API
+type CrunchyBridgeInventory struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   dbaasv1alpha1.DBaaSInventorySpec   `json:"spec,omitempty"`
+	Status dbaasv1alpha1.DBaaSInventoryStatus `json:"status,omitempty"`
+}
+
+//+kubebuilder:object:root=true
+
+// CrunchyBridgeInventoryList contains a list of CrunchyBridgeInventory
+type CrunchyBridgeInventoryList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []CrunchyBridgeInventory `json:"items"`
+}
+
+func init() {
+	SchemeBuilder.Register(&CrunchyBridgeInventory{}, &CrunchyBridgeInventoryList{})
+}
+
+type DatabaseProvider struct {
```

Comment:
Where/why do you need this? You should be able to use it from dbaas operator

Reply:
Removed. and not required here.